StarWarsExplorer.js

```diff
@@ -0,0 +1,490 @@
+const readline = require('readline-sync');
+
+//testing out import features
+const mapBuild = require('./supportFiles/maps');
+// const directions = require('./directions');
+const thePlayer = require('./supportFiles/playerStats');
+
+// initialize starting location in map
+let initialX;
+let initialY;
+let isNewStart;
+let shipRepaired;
+
+const startGame = () => {
+    readline.setDefaultOptions({limit: null});
+    isNewStart = true;
+    shipRepaired = false;
+    console.clear();
+    // Set starting location on the map
+    initialX = 0;
+    initialY = 0;
+    menuOptions = [];
+    menuOptions[1] = "New Game";
+    menuOptions[2] = "Load Game(Work in Progress)";
+    menu();
+}
+const menu = () => {
+    // Only place that asks for a number
+    while (menuOptions != 0) {
+        let gameMenu = readline.keyInSelect(menuOptions, 'Please choose your option',{cancel: 'Quit Game'});
+        switch (gameMenu) {
+            case 1:
+                if (isNewStart === true) {
+                    gameLoop();
+                    break;
+                } else {
```

Comment:
YOu're calling gameLoop() regardless of if or else. Why have the condition?

Reply:
This is section was because of the Menu feature in the game when you type Menu you resume the game from where you left off. the full feature wasn't implemented yet.